config/routes.rb

```diff
@@ -5,6 +5,7 @@
   delete '/logout', to: 'session#destroy'
   resources :users, only: [:new, :index, :create, :show]
   resources :items, only: [:index]
-  resources :categories, only: [:show, :index]
+  resources :categories, only: [:index]
+  get '/categories/:slug', to: 'categories#show', as: "category"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.